main.go

```diff
@@ -2,31 +2,90 @@ package main
 
 import (
 	"context"
+	"encoding/base64"
+	"net/http"
 	"os"
+	"strconv"
+	"time"
 
+	"github.com/asecurityteam/nexpose-vuln-notifier/pkg/assetfetcher"
+	nexposevulnnotiifier "github.com/asecurityteam/nexpose-vuln-notifier/pkg/handlers/v1"
+	"github.com/asecurityteam/runhttp"
 	serverfull "github.com/asecurityteam/serverfull/pkg"
 	serverfulldomain "github.com/asecurityteam/serverfull/pkg/domain"
 	"github.com/asecurityteam/settings"
+	"github.com/asecurityteam/transport"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
 func main() {
 	ctx := context.Background()
-	var _ lambda.Handler = nil // Placeholder to keep lambda imported. Delete after adding to the map.
-	handlers := map[string]serverfulldomain.Handler{
-		// TODO: Register lambda functions here in the form of
-		// "name_or_arn": lambda.NewHandler(myHandler.Handle)
+
+	nexposeHTTPClient := nexposeHTTPClient()
+
+	pageSize, _ := strconv.Atoi(os.Getenv("NEXPOSE_SITE_ASSET_RESPONSE_SIZE"))
```

Comment:
It seems like `NEXPOSE_SITE_ASSET_RESPONSE_SIZE` should be called `NEXPOSE_SITE_ASSET_PAGE_SIZE`.